### PR TITLE
feat(cli): add web app support cli

### LINKED ISSUE #114: Integrate web prover into mopro-cli
It'd be nice to be able to add `web` as a platform

And then have a basic template for this:

https://github.com/oskarth/mopro/tree/main/web-prover

Would make it more `mopro` complete offering for all platforms.

(Later on we can improve on it with PWA etc but start simple for now)

cc @vivianjeng

### PR DESCRIPTION
**Pull request by vivianjeng**
Initialize and build a web app.

```sh
mopro init --platforms web
cd mopro-example-app
mopro prepare
mopro build --paltforms web
```

Open web project directory and run frontend locally.

```sh
cd web
npm install
npm run dev
```

close #114

### CODE REVIEW

File: templates/mopro-example-app/web/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Web Prover
+
+This is used to benchmark proving time in browser with `snarkjs`.
+See: https://mopro-webprover.pages.dev/
+
+Current proofs:
```
Comment (oskarth) [inline templates/mopro-example-app/web/README.md:6]: I guess we'd want this to be just one circuit to map to other ones? But this works too to start with

File: mopro-cli/README.md [hunk 2]
```diff
@@ -88,6 +88,25 @@ mopro build --platforms android
...
 
+### Web
+
+Initialize and build a web app.
+
+```sh
+mopro init --platforms web
+cd mopro-example-app
+mopro prepare
+mopro build --paltforms web
```
Comment (oskarth) [inline mopro-cli/README.md:99]:
```suggestion
mopro build --platforms web
```

File: scripts/cli/build_web.sh [hunk 3]
```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Script for initializing and updating an Android (simplified) project with Rust bindings.
```
Comment (oskarth) [inline scripts/cli/build_web.sh:3]: Web

**Review by oskarth — APPROVED** [review]
Awesome, that was quick 👍

File: scripts/cli/build_web.sh [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    PROJECT_DIR=$(pwd)
+    TARGET_DIR=${PROJECT_DIR}/target
+    WEB_APP_DIR=${PROJECT_DIR}/web
+    
+    initialize_environment "$@"
+    read_configuration "$1"
+    
+    copy_circuit_files
+    
+    print_action "Done! Please open the web app with \`cd web && npm install && npm run dev\`"
```
Comment (oskarth) [inline scripts/cli/build_web.sh:63]: Should `build` for web also do npm build maybe? Do the circuit files have to be copied every build?

 I guess can tweak later on too

**Review by oskarth — APPROVED** [review]

File: scripts/cli/build_web.sh (hunk 4 above)
Comment (vivianjeng) [reply scripts/cli/build_web.sh]: actually no need for `npm build`
but it is used to copy circuits files (wasm and zkey)

**Review by oskarth — APPROVED** [review]

**Pull request merged**